### ISSUE TITLE
how to access BUILD_TARGET in babel.config.js

### ISSUE THREAD

**Issue by hardfist** [jaredpalmer/razzle#892]
in order to support `babel-plugin-import` plugin ,I need to give different option depends on BUILD_TARGET
```
## for server
['import', { libraryName: 'antd'}]
## for client
['import',{libraryName: 'antd', style: true}]
```
but it seems that process.env.BUILD_TARGET is undefined in babel.config.js, how can I use BUILD_TARGET in babel.config.js

**Comment by revskill10** [comment]
@hardfist I made a PR to do that [here](https://github.com/jaredpalmer/razzle/pull/930)

**Comment by nimaa77** [comment]
fixed in #1108
